visualizer/exampleGenerationRequests.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-env browser */
+
+'use strict';
+
+// TODO: handle invalid grammar in textbox
+
+(function(root, initModule) {
+  if (typeof exports === 'object') {
+    module.exports = initModule;
+  } else {
+    initModule(root.ohm, root.ohmEditor, root.exampleWorkerManager,
+               root.utils, root.domUtil, root.autosize);
+  }
+})(this, function(ohm, ohmEditor, exampleWorkerManager, utils, domUtil, autosize) {
+
+  var neededExamples = [];
+
+  var focusedElement = null;
+  var focusedRuleName = '';
+
+  // TODO: clicking off of textarea triggers refresh, deleting content
+
+  // logic for arrow to open drawer and show needed examples menu
+  var examplesNeededDrawer = domUtil.$('#examplesNeededDrawer');
+  var examplesNeededIndicator = domUtil.$('#examplesNeededDrawer .indicator');
+  var exampleSplitter = domUtil.$('#exampleSplitter');
+  examplesNeededDrawer.addEventListener('click', function() {
+    var ExampleRequestContainer = domUtil.$('#ExampleRequestContainer');
```

Comment:
Nit: Any reason this id and variable begin with a capital letter? If not, should be lowercase.